admin_guide/high_availability.adoc

```diff
@@ -153,29 +153,53 @@ $ ping hello-openshift.shard1.v3.rhcloud.com
 [[configuring-ip-failover]]
 == Configuring IP Failover
 
-Using IP failover involves switching IP addresses to a redundant or stand-by
-set of nodes on failure conditions.
+An IP Failover configuration is created using the `oadm ipfailover` command with suitable 
+xref:../admin_guide/high_availability.adoc#options-environment-variables[options].
+Internally, `oadm ipfailover` creates a DC (Deployment Config) based on provided options.
 
 [IMPORTANT]
 ====
-At this time of writing, ipfailover is not compatible with cloud
+Make sure that the service that relies on a VIP is running on every node in the
+set of IP Failover nodes. You never know which of the nodes will be servicing the
+VIP at any point in time.
```

Comment:
Yeah, this recommendations does have ramifications - see above.
There are times when you want some hot standby nodes to scale up/down based on load/failure.